.travis.yml

```diff
@@ -15,8 +15,8 @@ git:
 
 env:
   global:
-    - BRANCH=`git log --format="%H" -n 1`
-    - DIFF=`git diff --name-only --diff-filter=b HEAD...$BRANCH`
+    - LOG=`git log -n 1 | grep Merge`
+    - DIFF=`git diff --name-only --diff-filter=b ${LOG:7:8}...${LOG:16:8}`
```

Comment:
I am a bit worried this is fragile. $LOG looks like: `Merge: 5cb84ac4 5de8828b` so would something like this work better?
```
OLD=`echo $LOG | cut -d'  ' -f2`
NEW=`echo $LOG | cut -d'  ' -f3`
```

Reply:
Thank you for the suggestion